qa/tasks/ceph.py

```diff
@@ -75,8 +76,16 @@ def generate_caps(type_):
 @contextlib.contextmanager
 def ceph_crash(ctx, config):
     """
-    Gather crash dumps from /var/lib/crash
+    Gather crash dumps from /var/lib/ceph/crash
     """
+
+    # Add logs directory to job's info log file
+    with open(os.path.join(ctx.archive, 'info.yaml'), 'r+') as info_file:
+        info_yaml = yaml.safe_load(info_file)
+        info_file.seek(0)
+        info_yaml['archive']['crash'] = '/var/lib/ceph/crash'
```

Comment:
```
2020-08-17T16:33:02.566 ERROR:teuthology.contextutil:Saw exception from nested tasks
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/contextutil.py", line 31, in nested
    vars.append(enter())
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-08-17-1905/qa/tasks/ceph.py", line 162, in ceph_log
    info_yaml['archive']['log'] = '/var/log/ceph'
KeyError: 'archive'
2020-08-17T16:33:02.567 ERROR:teuthology.run_tasks:Saw exception from tasks.
```

could you ensure that `'archive' in info_yaml` before accessing it with an index?

Reply:
Done. Please see #36718